fotoapparat/src/main/java/io/fotoapparat/parameter/range/Range.java

```diff
@@ -0,0 +1,36 @@
+package io.fotoapparat.parameter.range;
+
+/**
+ * Created by ychen on 5/25/2017.
+ */
+
+public abstract class Range<T> {
+    abstract public boolean contains(T value);
+    abstract public T highest();
+    abstract public T lowest();
+
+    public String toString() {
+        return String.format("%1$s to %2$s", lowest(), highest());
+    }
+
+    public static <T> Range<T> emptyRange() {
```

Comment:
JavaDoc is missing. I would also name it `empty()` (because it is already clear from invocation that it is a range: `Range.emptyRange()`). But +1 for creating this.